xml/admin_gui_dashboard.xml

```diff
@@ -90,45 +96,39 @@
   <sect2 xml:id="dashboard-util-menu">
    <title>Utility Menu</title>
    <para>
-    The top right part of the screen contains a utility menu. It includes
+    The top right-hand side of the screen contains a utility menu. It includes
     general tasks related more to the dashboard than to the &ceph; cluster. By
-    clicking its items, you can access the following topics:
+    clicking the options, you can access the following topics:
    </para>
    <itemizedlist>
     <listitem>
      <para>
-      Change the language of the dashboard's user interface. You can choose
-      from Czech, English, German, Spanish, French, Portuguese, Chinese, or
-      Indonesian.
+      Change the dashboard's language interface to: Czech, German, English,
+      Spanish, French, Indonesian, Italian, Japanese, Korean, Polish,
+      Portuguese (Brazilian), and Chinese.
      </para>
     </listitem>
     <listitem>
      <para>
-      Display a list of &ceph; related tasks that are running in the
-      background.
+      Tasks and notifications
      </para>
     </listitem>
     <listitem>
      <para>
-      View and erase recent dashboard notifications.
+      View the documentation, information about the REST API, or further
+      information about the dashboard.
      </para>
     </listitem>
     <listitem>
      <para>
-      Display a list of links that refer to the information about the
-      dashboard, its complete documentation, and an overview of its REST API.
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      Manage the dashboard's users and user roles. Refer to
+      User management and telemetry configuration. Refer to
       <xref linkend="dashboard-user-roles" /> for more detailed command line
```

Comment:
Hmm, the user management is not really about command line usage. I'd suggest to separate this. We have CLI-based commands for performing dashboard user management tasks, but the menu item is about the built-in user management.

Reply:
It is indeed a confusion of the two, good spot. It should be separted into "This is for the dashboard. But if you want to manage using the CLI, see..."